experiments/jobs/BackUpOptimizely.groovy

```diff
@@ -0,0 +1,79 @@
+/*
+Pulls current state of experiment code in Optimizely and backs up to git history branch in edx/optimizely-experiments.
+
+Vars consumed for this job:
+    * FOLDER_NAME: folder
+    * NOTIFY_ON_FAILURE: alert@example.com
+    * OPTIMIZELY_TOKEN: optimizely api token (required)
+    * SECURE_GIT_CREDENTIALS: secure-bot-user (required)
+    * SSH_AGENT_KEY : ssh-credential-name
+*/
+package experiments.jobs
+
+class BackUpOptimizely {
+
+    public static def job = { dslFactory, extraVars ->
+
+                dslFactory.job(extraVars.get("FOLDER_NAME","Experiments") + "/back-up-optimizely") {
+
+                    def gitCredentialId = extraVars.get('SECURE_GIT_CREDENTIALS','')
+
+                    wrappers {
+                        credentialsBinding {
+                            string('OPTIMIZELY_TOKEN', 'optimizely-token')
+                        }
+                        sshAgent(gitCredentialId)
+                      }
+
+                    extraVars['NOTIFY_ON_FAILURE'] = 'revenue-squad-alert@edx.opsgenie.net'
```

Comment:
This should probably be in edx-internal and hidden so that the email that pages your team isn't publicly visible.